bin/gwdetchar-scattering

```diff
@@ -497,6 +501,15 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
 # close panel group
 page.div.close()
 
+# write a csv file.  
+logger.debug('Writing a summary CSV record')
+
+actives = actives.coalesce() # merge contigous segments 
+gps = [x for x in highsnrtrigs[names[0]] for y in actives if x in y]
+segs =  [y for x in highsnrtrigs[names[0]] for y in actives if x in y]
```

Comment:
@siddharth101, can you give this a shot?

```suggestion
segs =  [tuple(y) for x in highsnrtrigs[names[0]] for y in actives if x in y]
```

I have a hunch this might fix the bug you're seeing.

Reply:
Will try it.